README.md

```diff
@@ -111,6 +111,39 @@ for epoch in range(num_epochs):
     reader.reset() # Return to the beginning of dataset.
 ```
 
+### Reading Images as NumPy Arrays
+
+MLIO supports reading image files in jpeg, png, and recordio formats, using the [`ImageReader`](doc/python/data_reader.md#ImageReader) class. The code snippet below shows how you can quickly iterate through an image dataset in mini-batches.
+
+```python
+import numpy as np
+import mlio
+
+from mlio.integ.numpy import as_numpy
+
+file_dir = '/path/to/image_dir' # This can be a directory or a single file.
+dataset = mlio.list_files(file_dir, pattern='*.jpg') # This can be jpg, png, or rec, depending on the image format.
+
+# Setting general DataReader parameters.
+data_reader_params = mlio.DataReaderParams(dataset=dataset, batch_size=200)
+
+# Setting ImageReader parameters. Set img_frame to mlio.ImageFrame.RECORDIO for recordio datasets.
```

Comment:
recordio is not a wellknown encoding for people who have not used MXNet before. Saying `for MXNet RecordIO datasets.` would make it more clear.